docs/lovelace_card_types.md

```diff
@@ -25,7 +25,7 @@ These cards are WIP and things may change.
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
 | type | string | **Required** | `entity-filter`
-| filter | list | **Required** | See filter description
+| filter | list | **Required** | The filter can either be specified as YAML, as described in the Filter options, or as a list of Python dicts.  
```

Comment:
This is incorrect. It's not Python dicts. You can define valid JSON inside YAML, and so that's what it is.

Reply:
OK, I'll fix this. Is the proper technical term "a list of JSON objects"?

Reply:
Yes. I don't know if it makes sense to call it out as it's just part of YAML. I would probably avoid it, as it will confuse beginners of what the correct syntax is.

Reply:
I see your point in avoiding stating the obvious if it's part of the YAML specs. But I didn't know it about it and it might be that others dont' either and it would be a shame if they wouldn't be able to take full advantage of Home Assistant because they haven't read up on YAML.

What about not changing anything here and just give the example I added to `lovelace_index.md` ,where several JSON objects are listed?

Reply:
I think I prefer to keep it out of the docs. We should be consistent in our config formatting. It's already a lot of config, we don't have to make it more confusing.